src/main.js

```diff
@@ -1,17 +1,20 @@
 import { ValidatorPollingService } from './validator-polling-service.js';
-import { SMSNotifier } from './notifiers/index.js';
+import { SMSNotifier, TelegramNotifier } from './notifiers/index.js';
 import { config } from './config.js';
 import { BeaconAPIClient } from './beacon-api-client.js';
 import { validatorBalanceReducedAlert, validatorStatusChangedAlert } from './alerts/index.js';
 import { logger } from './logger.js';
 
-const smsNotifer = new SMSNotifier(config.sms)
+const smsNotifier = new SMSNotifier(config.sms);
+const telegramNotifier = new TelegramNotifier(config.telegram);
 const beaconApiClient = new BeaconAPIClient(config.beaconAPIs);
 const validatorPollingService = new ValidatorPollingService(beaconApiClient);
 
 validatorPollingService.addValidators(config.pubkeys);
-validatorPollingService.addListener(validatorBalanceReducedAlert(smsNotifer, config.alerts.validatorBalanceReduced));
-validatorPollingService.addListener(validatorStatusChangedAlert(smsNotifer));
+validatorPollingService.addListener(validatorBalanceReducedAlert(smsNotifier, config.alerts.validatorBalanceReduced));
+validatorPollingService.addListener(validatorBalanceReducedAlert(telegramNotifier, config.alerts.validatorBalanceReduced));
+validatorPollingService.addListener(validatorStatusChangedAlert(smsNotifier));
+validatorPollingService.addListener(validatorStatusChangedAlert(telegramNotifier));
```

Comment:
if we do go with the above suggestion, then: 

```suggestion
notifiers.forEach((notifier) => {
  validatorPollingService.addListener(validatorBalanceReducedAlert(notifier, config.alerts.validatorBalanceReduced));
  validatorPollingService.addListener(validatorStatusChangedAlert(notifier));
});
```